app/src/main/java/com/bignerdranch/android/geoquiz/MainActivity.kt

```diff
@@ -132,6 +140,12 @@ class MainActivity : AppCompatActivity() {
         toast.show()
     }
 
+    fun isAnswered () {
```

Comment:
two things here: 
- you have a space between `isAnswered` and `()`.
- `isAnswered()` by convention suggests that this method returns a `Boolean`, while in your implementation its a `Unit` type of method. You are also setting the `isEnabled` property of two buttons with a new value. Better to rename that method to `setIsAnswered()`. Better yet, create an extension function of `Button`, i.e `fun Button.setIsAnswered(isAnswered: Boolean)` and use it accordingly

Reply:
I renamed this method to `updateButtonsEnabledState` which is more precise